docs/installation.rst

```diff
@@ -0,0 +1,217 @@
+
+.. code-block:: shell
+
+    $ sky check
+
+This will show a detailed status of which cloud providers have already been configured properly.
+
+AWS
+---
+
+Start by installing the AWS CLI. Follow this
+`Getting Started <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>` guide
+or use ``pip install "runhouse[aws]"`` to include it with the Runhouse library.
+
+Next, configure AWS with the following command:
+
+.. code-block:: shell
+    $ aws configure
+
+You'll be prompted to enter your AWS Access ID. This can be found by logging into
+the `AWS Console <https://console.aws.amazon.com/console/home>`. Click on the name
+of your account in the top-left corner of the screen and then select
+"Securtiy credentials" from the dropdown.
+
+To verify that credentials are properly set up, run the SkyPilot command again:
+
+.. code-block:: shell
+
+    $ sky check
+
+For more info: `SkyPilot | AWS <https://skypilot.readthedocs.io/en/latest/cloud-setup/cloud-permissions/aws.html>`
+
+GCP
+---
+
+Begin by installing Runhouse with GCP:
+
+.. code-block:: shell
+
+    $ pip install "runhouse[gcp]"
+
+Additionally, you'll need to install GCP tools. Run the following commands:
+
+.. code-block:: shell
+
+    $ pip install google-api-python-client
+    $ conda install -c conda-forge google-cloud-sdk -y
+
+Your GCP credentials may also need to be set:
+
+.. code-block:: shell
+
+    $ gcloud init
+    # Run this if you don't have a credentials file.
+    $ gcloud auth application-default login
+    $ sky check
+
+
+You'll be prompted to pick a cloud project to use after running ``gcloud init``.
+If you don't have one ready yet, you can connect one later by listing your projects
+with ``gcloud projects list`` and setting one
+with ``gcloud config set project <PROJECT_ID>``.
+
+For more info: `SkyPilot | GCP <https://skypilot.readthedocs.io/en/latest/getting-started/installation.html#google-cloud-platform-gcp>`
+
+Azure
+-----
+
+If you have an Azure account, use the following commands to install Runhouse and configure your
+credentials to launch instances on their cloud.
+
+.. code-block:: shell
+
+    $ pip install "runhouse[azure]"
+    # Login
+    $ az login
+    # Set the subscription to use
+    $ az account set -s <subscription_id>
+    $ sky check
+
+For more info: `SkyPilot | Azure <https://skypilot.readthedocs.io/en/latest/getting-started/installation.html#azure>`
+
+
+Other Providers
+---------------
+
+For a full list of providers and configuration details: `SkyPilot documentation <>`.
+Some additional providers supported by Runhouse via SkyPilot include:
+
+- Kubernetes
+- Lambda Cloud
+- & more
+
+
+Cluster Lifecycle Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To ensure that you have full control over the availability of your clusters, and the cost associated
+with maintaining them, you can may find the following SkyPilot commands helpful:
+
+- ``sky status`` - Displays a list of all your clusters by name.
```

Comment:
thoughts on also linking to the sky API docs for each command, or you think thats just information overload?

Reply:
Good point - maybe just a link to https://skypilot.readthedocs.io/en/latest/reference/cli.html#cluster-cli